exercises/beer-song/src/Example.hs

```diff
@@ -30,5 +30,5 @@ verse n = unlines
   [ capitalize (stanza n) ++ " on the wall, " ++ stanza n ++ "."
   , action n ++ ", " ++ stanza (pred n) ++ " on the wall." ]
 
-sing :: Int -> Int -> String
-sing from to = unlines $ map verse [from, from - 1 .. to]
+sing :: String
```

Comment:
should we take the opportunity to change to `song` to match food-chain's https://github.com/exercism/xhaskell/blob/master/exercises/food-chain/src/Example.hs#L17 ?